packages/contracts-bedrock/contracts/test/OptimismMintableTokenFactory.t.sol

```diff
@@ -2,7 +2,7 @@
 pragma solidity 0.8.10;
 
 import { Bridge_Initializer } from "./CommonTest.t.sol";
-import { LibRLP } from "./Lib_RLP.t.sol";
+import { LibRLP } from "./RLP.t.sol";
```

Comment:
Probably should do this then:

```suggestion
import { RLP } from "./RLP.t.sol";
```

Reply:
I believe LibRLP is an external library

Reply:
Was taken from here I think https://github.com/Rari-Capital/solmate/issues/207

Needs to be attributed though, idk why it doesn't have any attribution attached

Reply:
Agree about attribution. The prefix is not a big deal to me tho, do what you want here. :)

Reply:
Attribution: https://github.com/ethereum-optimism/optimism/pull/2927